src/twisted/python/test/test_url.py

```diff
@@ -793,6 +799,25 @@ def test_technicallyTextIsIterableBut(self):
             URL(path=u'foo')
         self.assertEqual(
             str(raised.exception),
-            "expected iterable of text for path, got text itself: {}"
+            "expected iterable of text for path, not: {}"
             .format(repr(u'foo'))
         )
+
+
+class URLDeprecationTests(SynchronousTestCase):
+    """
+    L{twisted.python.constants} is deprecated.
+    """
+    def test_urlDeprecation(self):
```

Comment:
Is this test working? I am trying to understand why ? since the URL should be already imported earlier